example/mysql-image/run

```diff
@@ -0,0 +1 @@
+docker run -p3306:3306 -e MYSQL_ROOT_PASSWORD=mysecretpassword mysql-grpc-connector-database
```

Comment:
add execution bit. 
when you are running build, you don't create above image you create the image: example/postgres-grpc-connector-database:latest change one of them